From c5247e09244443d102792c9c7120394058bd2d06 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Thu, 26 Jul 2018 17:17:29 +0300
Subject: [PATCH 0820/1200] mvebu: config: update default bootcmd

Default bootcmd use kernel_addr, ramfs_addr and fdt_addr.
Mvebu default settings use kernel_addr_r, ramdisk_addr_r and
and fd_addr_r.
This cause incorrect usage of the addresses of the kernel,
ramdisk and fdt.

This patch update bootcmd to use the naming used in default
MVEBU settings.

Change-Id: I6f175b9f4fbee5f97e389b754ccc55da55bbe106
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/58457
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
---
 include/configs/mvebu_armada-common.h | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index 5f7d0a2e3d..1f0d229666 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -34,7 +34,12 @@
 #define CONFIG_ETHPRIME			"eth0"
 #define CONFIG_ROOTPATH                 "/srv/nfs/" /* Default Dir for NFS */
 #define CONFIG_ENV_OVERWRITE		/* ethaddr can be reprogrammed */
-#define CONFIG_EXTRA_ENV_SETTINGS	"kernel_addr_r=0x7000000\0"	\
+#define CONFIG_EXTRA_ENV_SETTINGS	"bootcmd=run get_images; " \
+						"run set_bootargs; " \
+						"booti $kernel_addr_r " \
+						"$ramdisk_addr_r " \
+						"$fdt_addr_r\0" \
+					"kernel_addr_r=0x7000000\0"	\
 					"initrd_addr=0xa00000\0"	\
 					"initrd_size=0x2000000\0"	\
 					"fdt_addr_r=0x6f00000\0"	\
-- 
2.22.0

